docs/reference/api/pages.md

```diff
@@ -0,0 +1,511 @@
+  // Content properties, e.g., `category`, `mainContent`
+  _targetId: 'ckhdscx5900054z9k88uqs16w',
+  _position: 'firstChild'
+};
+// Request inside an async function.
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page?apikey=myapikey', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify(data)
+});
+```
+
+### Response
+
+The successful POST request returns the newly created document. See the [page document response example](#page-document-response-example) below for a sample response body.
+
+## `PUT: /api/v1/@apostrophecms/page/:id`
+
+### Required properties
+
+| Property | Type | Description |
+|----------|------|-------------|
+|`_targetId` | String | The `_id` of an existing page to use as a target when inserting the new page|
+|`_position` | String | `before`, `after`, `firstChild`, or `lastChild`. This sets the position within the page tree for the new page in relation to the target page (see `_targetId`). `before` and `after` insert the new page as a sibling of the target. `firstChild` and `lastChild` insert the new page as a child of the target.|
+
+The `_position` property uses specific string values rather than index numbers to better support the draft review workflow.
+
+### Request example
+
+```javascript
+// Object with, at a minimum, properties for each required piece field.
+const data = {
+  title: 'My great new article',
+  // Content properties, e.g., `category`, `mainContent`
+  _targetId: 'ckhdscx5900054z9k88uqs16w',
+  _position: 'lastChild'
+};
+// Request inside an async function.
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey', {
+  method: 'PUT',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify(data)
+});
+```
+
+### Response
+
+The successful PUT request returns the newly created document. See the [page document response example](#page-document-response-example) below for a sample response body.
+
+## `PATCH: /api/v1/@apostrophecms/page/:id`
+
+**Authentication required.**
+
+The PATCH request may include *both* `_targetId` and `_position` as described in the [POST request description](#post-api-v1-apostrophecms-page), but that is not required if the page is not being moved.
+
+### Request example
+
+```javascript
+// Object with *only* the document fields to overwrite.
+// This example only changes the page's title to "Organization history."
+const data = {
+  title: 'Organization history'
+};
+// Request inside an async function.
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey', {
+  method: 'PATCH',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify(data)
+});
+```
+
+:::tip
+As a convenience, you may make a PATCH request for any MongoDB document, regardless of type using a catch-all route using the document's `_id` property:
```

Comment:
For any Apostrophe document. Not any MongoDB document (there are plenty of Mongo collections, maybe, in this app and the catch-all route won't know what that's about).